zenoh/src/lib.rs

```diff
@@ -138,9 +138,9 @@ pub mod core {
 }
 
 pub mod logging {
-    #[cfg(feature = "internal")]
-    pub use zenoh_util::{init_log_with_callbacks, try_init_logging};
     pub use zenoh_util::{init_logging, init_logging_with_level, InvalidLogLevel, LogLevel};
+    #[cfg(feature = "internal")]
```

Comment:
- better to use `#[zenoh_macros::internal]`
- the function implementations should be under "internal" too

Reply:
> better to use #[zenoh_macros::internal]

Indeed

> the function implementations should be under "internal" too

Do you mean moving `zenoh::logging::try_init_logging` to `zenoh::internal::try_init_logging`?

Reply:
No, I mean the implementation of internal/unstable functions are usually under "internal"/"unstable" cfg too, not only reexports.
It would be nice to check such cases automatically - generate build error if there is some public item inside which is not publicly reexported

Reply:
Got it, I will add the feature to `zenoh_util`.